controllers/node.go

```diff
@@ -458,6 +463,15 @@ func deleteEgressGateway(w http.ResponseWriter, r *http.Request) {
 		logic.ReturnErrorResponse(w, r, logic.FormatError(err, "badrequest"))
 		return
 	}
+	var gateway models.EgressGatewayRequest
+	gateway.NetID = params["network"]
+	gateway.Ranges = node.EgressGatewayRanges
+	err = logic.ValidateEgressRange(gateway)
```

Comment:
why do we need validate check on delete req?

Reply:
removed in new commit